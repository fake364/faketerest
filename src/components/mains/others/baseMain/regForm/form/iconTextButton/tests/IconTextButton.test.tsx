import { shallow } from 'enzyme';
import React from 'react';
import IconTextButton from '../IconTextButtom';
import { FaAddressBook } from '@react-icons/all-files/fa/FaAddressBook';

describe('IconTextButton', () => {
  it('should render base button with icon and text', () => {
    const wrapper = shallow(
      <IconTextButton text={'Text'} Icon={FaAddressBook} onClick={jest.fn()} />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
