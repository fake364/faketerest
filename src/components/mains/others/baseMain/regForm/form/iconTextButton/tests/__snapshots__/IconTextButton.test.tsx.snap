// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconTextButton should render base button with icon and text 1`] = `
<BaseButton
  className="mt-[8px] self-stretch rounded-common_radius py-[8px]"
  onClick={[MockFunction]}
>
  <div
    className="flex items-center justify-evenly"
  >
    <FaAddressBook
      className="mr-[8px] text-[24px]"
    />
    <div>
      Text
    </div>
  </div>
</BaseButton>
`;
